### DISCUSSION TITLE
Is this project alive?

### DISCUSSION THREAD

**Discussion by santi-petersen** [WhatsApp/stickers#1000] [General]
@simonzhexu @amrit-1901, Is this project alive?
I haven't seen any activity from the administrators in a long time. They don't answer in Issues and Discussions and do not make changes to the code or new commits.
I think it's important to keep this project alive, at least as a way to report bugs, especially WhatsApp bugs directly related to third-party stickers, like importing stickers to WhatsApp. Today there is a critical bug in WhatsApp Beta [Android] #998 and there is no way to report it to the WhatsApp developers, only from this project but there is no response. I'm afraid that they are not even aware of the bug and that it could reach production.

**Comment by Korogba** [comment]
Hi @santi-petersen, apologies for the late response here. The linked issue has since been fixed as noted on the issue but this was not communicated on time. Please keep reporting any issues you find and we will try our best to respond on time next time
